src/components/SkeletonPage/SkeletonPage.scss

```diff
@@ -32,6 +34,45 @@ $action-text-spacing: rem(24px);
   margin-bottom: (-1 * spacing(extra-tight));
 }
 
+.TitleAndPrimaryAction {
+  display: flex;
+
+  @include page-content-when-partially-condensed {
+    display: block;
+  }
+}
+
+.Title {
+  flex: 1 1 0%;
+}
+
+.PrimaryAction {
+  align-self: stretch;
+
+  > * {
+    height: $primary-action-button-height;
+    min-width: $primary-action-button-width;
+  }
+
+  @include page-content-when-layout-stacked {
+    margin-top: spacing(base);
+    margin-bottom: (-1 * spacing(tight));
+  }
+
+  @include page-content-when-not-fully-condensed {
+    margin-top: $actions-vertical-spacing;
+    margin-bottom: -8px;
```

Comment:
I wonder if we can use spacing for this? I'm not positive but maybe something like
```scss
  ...
  magin-bottom: -#{spacing(tight)}
  ...
```

Reply:
I don't think you need the interpolation here, some parens would be enough `margin-bottom: `-(spacing(tight));`

Dang sass allowing functions with hypens, not just in the name but at the start of the name